content/developer/sdk/java/ContrastJenkinsPlugin.md

```diff
@@ -4,66 +4,63 @@ description: "Sample Jenkins plugin using the Contrast Java SDK"
 
-## Threshold Condition Settings
+You can add as many rules as you like. The plugin fails on the **first** bad condition and tells you on which condition it failed.
 
-These settings are for filtering trace results in order to verify conditions configured in the post-build action.
+>**Note**: Even if your build succeeds, the plugin fails the overall build if the test finds a bad condition.
 
-| Parameter                    | Description                                                              |
-|------------------------------|--------------------------------------------------------------------------|
-| TeamServer Organization Uuid | Organization Uuid of the configured user found in Organization Settings, or can be copied from the URL when viewing the home page in TeamServer. |
-| TeamServer Application Name  | Name of the application as listed on the `Applications` tab in TeamServer. <BR> This is used to filter results in the query for post-build actions. |
+<a href="assets/images/Jenkins_threshold_condition.png" rel="lightbox" title="Threshold condition configuration"><img class="thumbnail" src="assets/images/Jenkins_threshold_condition.png"/></a>
 
-### How Threshold Conditions Work In Post-Build Action
+### Threshold conditions in a Pipeline step
 
-* Add a count. The count is exclusive, so if you set a count for 5, it will fail on 6 or more vulnerabilities. This field is required.
-* Add a severity (Note, Low, Medium, High, or Critical). The plugin sets a filter in the API call for all vulnerabilities greater than or equal to this field.
-* Add a vulnerability type. The type is the name of a rule. Here you can specify a single rule to filter for. The plugin checks for the number of vulnerabilities with the rule type and compares it to the count.
+When you add a Pipeline step with the name `contrastVerification`, it follows the same principles as the post-build action but in a newer format for Jenkins 2.0 improvements.
 
-Severity and vulnerability type are not required, but suggested in order to narrow down your results.
+Pipeline configuration:
 
-You can add as many rules as you like. The plugin fails on the **first** bad condition in order, and will tell you which condition it failed on.
+```
+contrastVerification profile: 'Localhost', count: 10, rule: 'xss', severity: 'High'
+```
 
->**Note**: Even if your build succeeds, the plugin will fail the overall build if a bad condition is found.
+## Test for Vulnerabilities
 
-### Examples
+In order for the Jenkins plugin to get accurate information, you must add a unique identifier built from the Jenkins CI configuration as an agent property. The corresponding property for the Java agent is `contrast.override.appversion`. Also, the job name must match your application name or you must override your application name with another property to ensure that Contrast tests for the correct information.
```

Comment:
Do we want to use code snippet formatting here for these property names? (```) instead of ('). Unless you are trying to move away from that inline and only use it for larger blocks of code. Just a thought.